TAScheduler/views_methods.py

```diff
@@ -193,28 +194,61 @@ def __init__(self, course_info):
         self.course_database = course_info
 
     def addInstructor(self, active_instr):
-        pass
+        if not isinstance(active_instr, InstructorObj):
+            raise TypeError("active_instr is not an instance of InstructorObj")
+        if InstructorToCourse.objects.filter(course=self.course_database).count() >= self.course_database.max_instructors:
+            raise ValueError("This course has reached the maximum number of instructors")
+        if InstructorToCourse.objects.filter(
+                instructor=active_instr.database).count() >= active_instr.database.max_assignments:
+            raise ValueError("Instructor has reached the maximum number of course assignments")
+        InstructorToCourse.objects.create(instructor=active_instr.database, course=self.course_database)
 
     def addTa(self, active_ta):
-        pass
+        if not isinstance(active_ta, TAObj):
+            raise TypeError("active_ta is not an instance of TAObj")
+        if TAToCourse.objects.filter(course=self.course_database).count() >= self.course_database.max_tas:
```

Comment:
^ max_tas doesn't exist use "num_of_sections" *I think*

(my test's pass using this logic but that's how i framed the tests)